pkg/eval/mods/file/file_test.go

```diff
@@ -29,6 +29,25 @@ func TestFile(t *testing.T) {
 			errs.ArityMismatch{
 				What:     "arguments here",
 				ValidLow: 1, ValidHigh: 1, Actual: 0}),
+
+		That("file:prclose").Throws(
+			errs.ArityMismatch{
+				What:     "arguments here",
+				ValidLow: 1, ValidHigh: 1, Actual: 0}),
+
+		That("file:pwclose").Throws(
+			errs.ArityMismatch{
+				What:     "arguments here",
+				ValidLow: 1, ValidHigh: 1, Actual: 0}),
```

Comment:
Looking at these tests and the two above, I don't find them necessary.

The code for matching parameters and arguments is implemented generically for every function in https://github.com/elves/elvish/blob/master/pkg/eval/go_fn.go, there's no need to test it for every builtin function. I'll remove these test cases after merging.